src/main/java/meteordevelopment/meteorclient/systems/modules/player/AutoTool.java

```diff
@@ -173,12 +180,11 @@ public static double getScore(ItemStack itemStack, BlockState state, boolean sil
         if (itemStack.getItem() instanceof SwordItem item && (state.getBlock() instanceof BambooBlock || state.getBlock() instanceof BambooSaplingBlock))
             score += 9000 + (item.getMaterial().getMiningLevel() * 1000);
 
-
         return score;
     }
 
-    public static boolean isTool(ItemStack itemStack) {
-        return itemStack.getItem() instanceof ToolItem || itemStack.getItem() instanceof ShearsItem;
+    public static boolean isTool(Item item) {
+        return item instanceof ToolItem || item instanceof ShearsItem;
```

Comment:
Why did you change it to take in `Item`? From my perspective a useless change that only breaks compatibility with third party code.